org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/PreferenceManager.java

```diff
@@ -196,4 +196,10 @@ public void removePreferencesChangeListener(IPreferencesChangeListener listener)
 	public IEclipsePreferences getEclipsePreferences() {
 		return eclipsePrefs;
 	}
+
+	public void setDefaultPreferences() {
```

Comment:
initialize() already sets those prefs for the DefaultScope, which is called in the PreferenceManager constructor, so I don't see why we need this new method